IdentityServer/v7/docs/content/ui/server_side_sessions/session_management.md

```diff
@@ -19,6 +19,10 @@ The [session management service]({{<ref "/reference/services/session_management_
 
 The Quickstart UI contains a simple administrative page (under the "ServerSideSessions" folder) that uses the *ISessionManagementService* API.
 
+{{% notice note %}}
+The UI template doesn't include pre-configured authorization in the form of an authorization policy e.g. We strongly recommend to add authorization suitable to your organization for non-test environments.
```

Comment:
I think we should expand this a bit, and it also needs some copy-editing.

I would try to emphasize what the end user needs to do, rather than what the template doesn't do.

Reply:
I have changed the wording.